packages/select/src/index.mts

```diff
@@ -79,7 +84,9 @@ export default createPrompt<string, SelectConfig>((config, done) => {
     .map(({ name, value, disabled }, index) => {
       const line = name || value;
       if (disabled) {
-        return chalk.dim(`- ${line} (disabled)`);
+        return chalk.dim(
+          `- ${line} (${typeof disabled === 'string' ? disabled : 'disabled'})`
```

Comment:
```suggestion
          `- ${line} ${typeof disabled === 'string' ? disabled : '(disabled)'}`
```

I think that might be preferable so that we also allow customizing the `()` at the same time.